crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -171,6 +171,22 @@ def ceph_status
         ret
       end
 
+      def swift_check
+        # 1. number of replicas
+        ret = {}
+        prop = Proposal.where(barclamp: "swift").first
+        return ret if prop.nil?
+        replicas = prop["attributes"]["swift"]["replicas"]
```

Comment:
Are you sure? 
2 zones, 2 replicas and 2 swift-storage nodes works for me.
2 zones, 3 replicas, 2 swift-storage zones and I got an error before